### PR TITLE
Add mention of WordPad as 'unavailable' <Task 23495284>

### PR DESCRIPTION
**Pull request by kaushika-msft**
Add mention of WordPad as 'unavailable' in Server Core SKU (CELA compliance)

### CODE REVIEW

**Comment by eross-msft** [comment]
@kaushika-msft  If you can remove the HTML and update the text (see comment above), I can go ahead and merge this change.

**Comment by eross-msft** [comment]
#on-hold

**Comment by eross-msft** [comment]
@kaushika-msft  Please update this PR within the next 30 days or it will become necessary to close the PR.

File: WindowsServerDocs/administration/server-core/what-is-server-core.md [hunk 1]
```diff
@@ -75,10 +75,12 @@ The following table shows which applications are available *locally* on Server C
 | Mstsc.exe                          | not available   | available                      |
 | Remote Desktop Services            | available       | available                      |
 | Hyper-V Manager                    | not available  | available                      |
-
+| WordPad <sup>1</sup>               | not available  | available                      |
```
Comment (illfated) [inline WindowsServerDocs/administration/server-core/what-is-server-core.md]:
```suggestion
| WordPad\*               | not available  | available                      |
```

File: WindowsServerDocs/administration/server-core/what-is-server-core.md [hunk 2]
```diff
@@ -75,10 +75,12 @@ The following table shows which applications are available *locally* on Server C
...
 
 For more information about what *is* included in Server Core, see [Roles, Role Services, and Features included in Windows Server - Server Core](server-core-roles-and-services.md). And for information about what *is not* included in Server Core, see [Roles, Role Services, and Features not included in Server Core](server-core-removed-roles.md)
 
+<sup>1</sup> To read .rtf  files locally stored on Server Core SKU, users can copy to a different Windows computer where WordPad is present.
```
Comment (illfated) [inline WindowsServerDocs/administration/server-core/what-is-server-core.md]:
```suggestion
\* To read .RTF  files locally stored on a Server Core SKU, users can copy the file(s) to a different Windows computer where WordPad is present.
```

**Comment by eross-msft** [comment]
Closing PR due to inactivity.

**Comment by illfated** [comment]
@eross-msft : Would you like to have this content added in a new PR from me instead?

**Comment by eross-msft** [comment]
@illfated Feel free to create a new PR for this content; however, we need to pull out all of the HTML code unless it's in a table. If you don't want to deal with that, I was going to add this to my backlog and address it when we had a spare moment or two...totally your call. 
Separately, thanks again for everything you've been doing!

**Comment by illfated** [comment]
Fair enough, thank you and thanks also for letting me know the amount of work required. I'll see what I can do about it.

**Pull request closed without merging**